x-pack/plugins/maps/public/components/layer_panel/view.js

```diff
@@ -132,13 +133,7 @@ export class LayerPanel extends React.Component {
         <EuiFlyoutHeader hasBorder className="mapLayerPanel__header">
           <EuiFlexGroup responsive={false} alignItems="center" gutterSize="s">
             <EuiFlexItem grow={false}>
-              <EuiButtonIcon
```

Comment:
Why did you remove the button here? This is a feature change that should not be in a localization PR.